### PR TITLE
standardize .deb package output file name

### PR DESCRIPTION
**Pull request by AndySchroder**
Uses the standardized format `lpac_X.X.X_arm64.deb` instead of the ambiguous `lpac-X.X.X-Linux.deb`.

also prepares to fix issue #99

### CODE REVIEW

**Comment by AndySchroder** [comment]
Please close https://github.com/estkme-group/lpac/pull/108 first.  It will be much easier to see how compact this change is.

**Comment by damonto** [comment]
You should checkout a new branch on the branch of main.

File: .gitignore [hunk 1]
```diff
@@ -17,7 +17,7 @@ build
 /src/version.h
 
 # for package files
-lpac-*.deb
+lpac_*.deb
```
Comment (septs) [inline .gitignore:20]: why change filename pattern?

Comment (AndySchroder) [reply .gitignore:20]: Please see https://github.com/estkme-group/lpac/pull/109#issue-2324688290 and https://github.com/estkme-group/lpac/issues/99#issuecomment-2138031867

**Comment by AndySchroder** [comment]
> You should checkout a new branch on the branch of main.

done, awaiting your review

**Comment by AndySchroder** [comment]
Any reason why you don't want to merge this?

**Comment by BeRealQueally** [comment]
close as https://github.com/estkme-group/lpac/pull/110#issuecomment-2138686338

**Comment by AndySchroder** [comment]
> close due to [#110 (comment)](https://github.com/estkme-group/lpac/pull/110#issuecomment-2138686338)

Sorry, I did not understand the nomenclature of pull requests. I meant to say in that comment to please "merge" this first, not "close" it.

**Comment by AndySchroder** [comment]
Please reopen.

**Pull request closed without merging**